spec/controllers/webhooks_controller_spec.rb

```diff
@@ -124,6 +124,23 @@
 
     end
 
+    context 'with a change update event' do
+
+      let(:optin_json) { File.read("spec/fixtures/change_notification_request.json") }
```

Comment:
Right. There's this strange convention throughout the spec file where file name strings are defined with double quotes